test/Interop/Cxx/destructor/Inputs/basic.h

```diff
@@ -0,0 +1,28 @@
+#ifndef TEST_INTEROP_CXX_DESTRUCTOR_INPUTS_BASIC_H
```

Comment:
Similarly to the comments on test location in https://github.com/apple/swift/pull/32378, WDYT about `test/Interop/Cxx/value-witness-table` for the value witness table tests?

Reply:
Sure.